utils/build-script-impl

```diff
@@ -3122,7 +3122,7 @@ for host in "${ALL_HOSTS[@]}"; do
                     with_pushd ${results_dir} \
                         call "${llvm_build_dir}/bin/llvm-lit" \
                              "${lldb_build_dir}/lit" \
-                             ${LLVM_LIT_ARGS} \
+                             ${LLVM_LIT_ARGS} -v --time-tests \
```

Comment:
What about amending LLVM_LIT_ARGS only once on line 3116?

Reply:
or even earlier, since we probably want this to affect every lit invocation?

Reply:
got it.